main_test.go

```diff
@@ -49,3 +52,69 @@ func TestOptionsToSlice(t *testing.T) {
 	opt.sortKeys = false
 	assert.NotContains(t, opt.ToSlice(), "-S")
 }
+
+func TestDocumentReadFrom(t *testing.T) {
+	testMsg := "hello world"
+	testReader := strings.NewReader(testMsg)
+
+	doc := &Document{
+		options: Options{
+			command: "./testdata/cat",
+		},
```

Comment:
The entire `options` field can be omitted here since it's not used in `ReadFrom`